packages/twenty-front/src/modules/workflow/search-variables/utils/getVariableTag.ts

```diff
@@ -0,0 +1,50 @@
+import { Node } from '@tiptap/core';
+import { mergeAttributes, RawCommands } from '@tiptap/react';
+
+export const getVariableTag = () =>
```

Comment:
I'm not sure why Charles suggested using a `getVariableTag` function. It doesn't seem correct to me. `Node.create` is a constructor, and I think it would be better and more idiomatic to call it once. Do I miss something?